core/objects.go

```diff
@@ -58,6 +58,7 @@ const (
 )
 
 // These types are the available challenges
+// TODO(aaron): Make this a custom type as well.
```

Comment:
As of late we've generally tried to avoid these kinds of TODOs because they get easily lost, preferably create a ticket for the change and ref it, or make the change in this PR.

Reply:
Good call, done: #5009